src/ert/run_models/base_run_model.py

```diff
@@ -610,7 +615,7 @@ async def run_ensemble_evaluator_async(
         await evaluator._server_started
         if not (await self.run_monitor(ee_config, ensemble.iteration)):
             await evaluator_task
-            return []
+            return _UserCancelled()
```

Comment:
Is it a good practice to return Exceptions and raise it somewhere else?
Maybe it would be better to just do the extra logic already here, and raise it directly? 
```suggestion
            self.active_realizations = [False for _ in self.active_realizations]
            raise _UserCancelled()
```

Reply:
Agree with @jonathan-eq on this one. Besides, it might be that `await evaluator_task` raises first if something fails (?)

Reply:
it now raises instead

Reply:
added a nicer error message if it fails while waiting for evaluator_task